app/api.py

```diff
@@ -580,6 +625,60 @@ def channel_acl(self, id, channel_id):
         data = obj_to_dict(server.getACL(channel_id))
         return Response(json.dumps(data, sort_keys=True, indent=4), mimetype='application/json')
 
+    @conditional(auth.login_required, auth_enabled)
+    @route('<int:id>/channels/<int:channel_id>/acl', methods=['PATCH'])
+    def update_channel_acl(self, id, channel_id):
+        """ Update specific channel ACL
+        """
+
+        server = meta.getServer(id)
+
+        # Return 404 if not found
+        if server is None:
+            return jsonify(message="Not Found"), 404
+
+        origin_acl = server.getACL(channel_id)
+        update_acls = origin_acl[0]
+        update_groups = origin_acl[1]
+        update_inherit = origin_acl[2]
+
+        params = request.get_json()
+        if params['acls'] is not None:
+            new_acls = []
+            for props in params['acls']:
+                new_acls.append(Murmur.ACL(
+                    props['applyHere'],
+                    props['applySubs'],
+                    props['inherited'],
+                    props['userid'],
+                    props['group'],
+                    props['allow'],
+                    props['deny'])
+                )
+                update_acls = new_acls
+
+        if params['groups'] is not None:
+            new_groups = []
+            for props in params['groups']:
+                new_groups.append(Murmur.Group(
+                    props['name'],
+                    props['inherited'],
+                    props['inherit'],
+                    props['inheritable'],
+                    props['add'],
+                    props['remove'],
+                    props['members'])
+                )
+                update_groups = new_groups
+
+        if params['inherit'] is not None:
+            update_inherit = bool(params['inherit'])
```

Comment:
Can you provide a description and example of all the params being passed here? I'm not sure the best way of testing this.

Reply:
```
acls example
curl --location --request PATCH 'http://{ip}/servers/1/channels/5/acl' \
--header 'Content-Type: application/json' \
--data '{
    "acls": [
        {
            "allow": 0,
            "applyHere": false,
            "applySubs": true,
            "deny": 8,
            "group": "~sub,0,1",
            "inherited": false,
            "userid": -1
        },
        {
            "allow": 8,
            "applyHere": true,
            "applySubs": true,
            "deny": 0,
            "group": "in",
            "inherited": false,
            "userid": -1
        }
    ]
}'
```

Reply:
Params is check here [mumble.Group](https://github.com/mumble-voip/mumble/blob/f5489231a9d3d23a8bed241217c9968826ab45c7/src/murmur/Murmur.ice#L126)

```
curl --location --request PATCH 'http://{ip}/servers/1/channels/13/acl' \
--header 'Content-Type: application/json' \
--data '{
    "groups": [
        {
            "name": "test1",
            "inherit": true,
            "inheritable": true,
            "inherited": false,
            "add": [0, 1],
            "remove": [],
            "members": []
        }
    ]
}'
```

Reply:
Also I have an error on mumble.Group.
`member` is readonly is attribute of group, it can not be set.
I will commit new version to remove member from param of request.

Reply:
Ok, sounds good!